roles/preflight/tasks/main_standalone_containers_certification.yml

```diff
@@ -34,6 +34,18 @@
       loop: "{{ preflight_containers_to_certify }}"
       loop_control:
         loop_var: c
+      when:
+        - not do_container_parallel_test
+        - max_images_per_batch == 1
```

Comment:
there's a small chance someone passes -1 here and none of the checks run, change this to `max_images_per_batch <= 1`

Reply:
Thanks, Jorge!